src/classes/FileManager.java

```diff
@@ -0,0 +1,126 @@
+package classes;
+
+import java.io.*;
+import java.util.List;
+
+/**
+ * Created by aloginov on 14.09.16.
+ */
+public class FileManager {
+
+    private File file;
+    private BufferedReader in;
+    private BufferedWriter out;
+
+    public FileManager(String fileName){
+        this.file = new File(fileName);
+    }
+
+
+
+    public void openFileForInput(){
+
+        if (!isFileExists()){
+            createNewFile();
+        }
+
+        if (isFileExists()){
+            try {
+                this.in = new BufferedReader(new FileReader(file));
+            } catch (IOException ex){
+                System.err.println(ex);
+            }
+        } else
+            createNewFile();
+
+    }
+
+    public void openFileForOutput(){
+
+        if (!isFileExists()){
+            createNewFile();
+        }
+
+        if (isFileExists()){
+            try {
+                this.out = new BufferedWriter(new FileWriter(file));
+            } catch (IOException ex) {
```

Comment:
I fixed it in 'read' methods. But I don't think it a good idea to do it 'open' methods. As far as I understand 'finally' will work in both cases either try or catch section will be done. But if file will be openned, I will be closed immediately. So I will add 'finally' section with close() statement into 'read' methods: file has/has not been read and closed.